_posts/2024-02-01-sswg-update-2024.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+published: false
+date: 2024-02-01 10:00:00
+title: SSWG 2024 Annual Update
+author: [Joannis]
+---
+
+
+In this yearly post, the Swift Server WorkGroup (SSWG) reflects on the community, ecosystem wide accomplishments and the WorkGroup's focus areas for the year ahead.
+
+Since our [previous update post](/blog/sswg-update-2023/), Swift on the Server has continued to mature.
+
+Let's start with a review of the progress made in 2023, before looking at the next steps in 2024.
+
+# 2023 in Review
+
+## Continued focus on growing the ecosystem
+
+The SSWG has continued to focus on growing the ecosystem by incubating packages and providing guidance to package authors.
+
+Memcached, which been in development througn GSoC, has made a lot of progress. The wire protocol and higher level connection APIs are done, but more work is still needed. The SSWG is looking forward to continued work on this package.
+
+Swift Distributed Tracing has seen it's 1.0 release in the summer of 2023, and has bseen adoption by several packages such as Hummingbird, GRPC and MongoKitten. While much progress has been made, expanding and supporting adoption of this package is still a goal going into 2024.
+
+The standardised Connection Pool Library has seen a lot of progress, and is currently available under SPI as part of PostgresNIO. This is considered an alpha release, and will eventually be extracted into its own package as the connection pool matures.
+
+## Adoption of Structured Concurrency
+
+We've seen a lot of progress on the adoption of Structured Concurrency. Most new libraries are adopting it, and we're working on a guide to help existing libraries adopt it as well.
+
+Rewrites of Hummingbird and GRPC are under way that are implemented entirely with Structured Concurrency. Moving to structured concurrency allows NIO to become more of an implementation detail. In addition, structured concurrency provides many advantages to libraries and the adopters, which will be further elaborated on in our guides.
```

Comment:
I would mention `ServiceLifecycle` here as the first thing since we 2.0.0 it last year. Talking about how it shapes applications and fully embraces structured concurrency.